src/modules/navigator/rtl_params.c

```diff
@@ -116,3 +116,19 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * @group Return Mode
  */
 PARAM_DEFINE_INT32(RTL_TYPE, 0);
+
+/**
+ * Half-angle of the RTL cone.
+ *
+ * Defines the half-angle of the cone which defines the vehicle RTL behavior.
+ *
+ * @min 0
+ * @value 0 No cone, always climb to altitude of at least RTL_RETURN_ALT above home.
+ * @value 1 25 degrees cone half angle.
```

Comment:
You can still use `@value 25`, etc, avoiding the need for the `switch case`.

Reply:
I agree, just use a float param and call it `RTL_CONE_ANGLE`.

Reply:
@bkueng @julianoes Thanks for the review, I did as you suggested.